public/Electron/utils.js

```diff
@@ -65,6 +65,10 @@ const handleErrorMessage = error => {
       detail = "모듈정보를 찾을 수 없습니다.";
       break;
     }
+
+    if (lines[i].includes("npm ERR!")) {
+      detail = "npm ERR";
```

Comment:
detail은 사용자에게 보이는 문장이니까 약어보다는 풀네임으로 쓰는게 좋지 않을까요?
```suggestion
      detail = "npm ERROR";
```